tools/addPrefix/make_prefixable.py

```diff
@@ -0,0 +1,63 @@
+from argparse import ArgumentParser
```

Comment:
Why is the `make_prefixable` script included in this PR? Are you making use of it in the SHACL work? If not, we might want to separate this out into another PR so we can discuss it directly. The building models are going to be changed by another PR that updates them to the latest version of the ontology.

Reply:
Oh no I meant to move this set of files into a different branch.